src/main/java/com/anatawa12/forge/gradle/separated/SeparatedLauncher.java

```diff
@@ -32,7 +33,10 @@ public SeparatedLauncher arg(String path) {
     public void run(final Project project) {
         project.javaexec(javaExecSpec -> {
             javaExecSpec.classpath(project.getConfigurations().getByName(configurationName));
-            javaExecSpec.setMain("com.anatawa12.forge.gradle.separated." + name + ".Main");
+            String main = "com.anatawa12.forge.gradle.separated." + name + ".Main";
+            GradleVersionUtils.choose("6.4",
+                    () -> javaExecSpec.setMain(main),
+                    () -> javaExecSpec.getMainClass().set(main));
```

Comment:
there are many code similar to here so I think it's better to add `JavaExecSpecHelper` (class like `JavaExtensionHelper`) and calling `JavaExecSpecHelper.setMain(javaExecSpec, "com.anatawa12.forge.gradle.separated." + name + ".Main")` is better.